### PR TITLE
[Hub Generated][Do not Merge] Review request for Microsoft.PolicyInsights to add version preview/2017-08-09-preview

### PR DESCRIPTION
**Pull request by najagasi**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/najagasi/azure-rest-api-specs/dev-policyinsights-Microsoft.PolicyInsights-2017-08-09-preview...master).

Do not merge 

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5810 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5810/commits/dcda66f03cdc0a313fa2ea9f324c694475cf7514

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json
Comment (kpajdzik) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json]: Can this be enum constrained?

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json
Comment (kpajdzik) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json]: Same here - can this be enum constrained?

**Comment by kpajdzik** [comment]
LGTM. Waiting for ARM review.

**Comment by kpajdzik** [comment]
@najagasi, could you rebase this pull request? The diff makes it really hard to review.

**Comment by kpajdzik** [comment]
@najagasi, any updates here?

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json
Comment (kpajdzik) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json]: I'd format that like the rest of the examples.

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json
Comment (kpajdzik) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json]: Nit - I think that would look better formatted like other objects.

**Comment by najagasi** [comment]
I am waiting for some other code fixes before I can merge the PR.
I will open the PR again when those changes are in.

**Comment by ravbhatnagar** [comment]
Only change seems to be adding the pagination in both policyStates and policyEvents. Signing off from ARM side.

**Review by kpajdzik — APPROVED** [review]

**Comment by kpajdzik** [comment]
> I am waiting for some other code fixes before I can request for merging the PR.
> I will open the PR again when those changes are in.

Is it still not ready to be merged?

**Comment by kpajdzik** [comment]
> I am waiting for some other code fixes before I can request for merging the PR.
> I will open the PR again when those changes are in.

@najagasi, Can this be merged?

**Comment by kpajdzik** [comment]
@najagasi, ping

**Comment by kpajdzik** [comment]
Lack of activity. Please reopen when ready to merge.

**Pull request closed without merging**
